bika/lims/browser/worksheet/views/analyses.py

```diff
@@ -470,6 +470,16 @@ def get_slot_header_data(self, obj):
             parent_url = api.get_url(client)
             parent_img = "client.png"
             parent_img_text = t(_("Client"))
+            # parent
+            client_reference = request.getClientReference()
+            parent_obj = client_reference
+            parent_title = api.get_title(client_reference)
+            parent_url = api.get_url(client_reference)
```

Comment:
`client_reference` is an string (that in most cases is empty), not an object/brain/uid, so trying to get it's title or url does not make sense. For instance, look at the function `api.get_title`:

https://github.com/senaite/senaite.core/blob/d6b140da8e65785d65245cff0cc66858572ec3a0/bika/lims/api/__init__.py#L365-L375

Since the passed in param is neither a brain nor an object, the function will try to get the object `get_object` function and return it's title: `return get_object(brain_or_object).Title()`

But since `get_object` will not be able to resolve any object for this string:
https://github.com/senaite/senaite.core/blob/d6b140da8e65785d65245cff0cc66858572ec3a0/bika/lims/api/__init__.py#L225-L241

you will get an `APIError` exception similar to `foo is not supported`